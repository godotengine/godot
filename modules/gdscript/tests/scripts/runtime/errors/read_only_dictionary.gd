func test():
	var dictionary := { "a": 0 }
	dictionary.make_read_only()
	dictionary.a = 1
