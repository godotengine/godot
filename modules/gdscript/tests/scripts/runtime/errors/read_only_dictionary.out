GDTEST_RUNTIME_ERROR
>> SCRIPT ERROR
>> on function: test()
>> runtime/errors/read_only_dictionary.gd
>> 4
>> Invalid assignment on read-only value (on base: 'Dictionary').
